### ISSUE TITLE
Definition list: list not using semantic markup (1.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#842]
### Expected behavior

Definition lists use <dl> description list semantics to convey set of data and relationships between key and value. Screenreaders can use this structure.
### Actual behavior

All data is in divs with no relationships/structure.

**Comment by Blackbaud-PatrickOFriel** [comment]
Syntax: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

**Comment by Blackbaud-PatrickOFriel** [comment]
Using dl, dd, dt etc won't work because it is not legal (in html terms) to have other divs inside of a dl, which we need to allow to have our component working. Would it be sufficient to use something like `aria-describedby` to relate the label to the value @Blackbaud-MattGregg ?
